js/components/App.js

```diff
@@ -1,13 +1,27 @@
 import 'babel/polyfill';
 
+// Mutations
+import ToggleWidgetEnableMutation from '../mutations/ToggleWidgetEnableMutation';
+
 class App extends React.Component {
+  _handleToggleEnabled = (i) => {
```

Comment:
Would this not be nicer as `_handleToggleEnabled(i) {`? (or even remove the underscore)

Also, wouldn't it be easier to pass in the node instead of reaching in with the index?

ie, `this.handleToggleEnabled(edge.node)` below